sample/src/main/java/com/fueled/flowr/sample/HomeFragment.java

```diff
@@ -42,7 +42,7 @@ public void onClick(View view) {
     }
 
     private void displayViewFragment() {
-        getFlowr().open(ViewFragment.class)
+        getFlowr().open("/m/From%20HomeFragment")
```

Comment:
@chetan-fueled yes, those are not literal strings but URI represented in the form of a string, so it need to be url encoded.